doc/release/release_notes.rst

```diff
@@ -206,6 +206,15 @@ Release Notes
         Previously, it would sometimes fail to update minimal and maximum
         versions of the range in particular conditions.
 
+    .. change:: changed
```

Comment:
Unrelated to this ticket but the installer in general. I still think that the -r instead of a --target or --destination is not ideal, and now ran into this:

```
[wiz] la3d15 claudiaz/dev > wiz install /tmp/qip/definitions/vertov-0.4.0.json -r /tmp/
warning: Error occurred trying to load definition from '/tmp/rv_test/example-rv/usurp.json'
warning: Error occurred trying to load definition from '/tmp/rv_test/example-rv/cookiecutter.json'
warning: Error occurred trying to load definition from '/tmp/maya_test/example-maya/usurp.json'
warning: No changes detected in release.

[wiz] la3d15 claudiaz/dev > wiz install /tmp/qip/definitions/vertov-0.4.0.json -r /tmp/test_release
info: Successfully installed 1 definition(s) to registry '/tmp/test_release'.
```

It seems to take the -r as input as well as output?

Reply:
> It seems to take the -r as input as well as output?

No it doesn't, but it is treating "/tmp" as a registry and check for duplicated definition before installation, hence the warnings

Reply:
but it fails to release

Reply:
Because `/tmp/qip/definitions/vertov-0.4.0.json` is under `/tmp`! :)

Reply:
Ok I understand better your other issue,`-r` is used to set default registries and also by the install command to set targeted registry, which is confusing:

```
> wiz -r /input install -r /output
```

I'll replace the output by `-o` for clarity

Reply:
so much better! thanks! works!